lib/opensusebasetest.pm

```diff
@@ -727,6 +727,21 @@ sub wait_grub_to_boot_on_local_disk {
     my @tags = qw(grub2 tianocore-mainmenu);
     push @tags, 'encrypted-disk-password-prompt' if (get_var('ENCRYPT'));
 
+    # Enable boot menu for x86_64 uefi workaround, see bsc#1180080 for details
+    if (is_sle && get_required_var('FLAVOR') =~ /Migration/ && check_var('ARCH', 'x86_64') && get_var('UEFI')) {
+        if (!check_screen(\@tags, 15)) {
+            record_soft_failure 'bsc#1180080';
+            send_key "e";
+            send_key "f2";
```

Comment:
Several key sent continuous, could you use wait_sceen_change to send key so that to avoid machine is low cause can't receive key?